org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/Utils.java

```diff
@@ -66,4 +68,13 @@ public static void print(String a) {
 	public static void printDType(Dataset a) {
 		System.out.println(DTypeUtils.getDTypeName(a));
 	}
+	
+	public static TestWatcher testWatcherCreator()
+	{
+		return new TestWatcher() {
+			protected void starting(Description description) {
+				System.out.println("\nStarting example : " + description.getMethodName());
```

Comment:
Now that I can run multiple examples, can you print the classname and the method name of the test:

e.g. instead of:


```
Starting example : lazyLarge1
The image has shape [2048, 2048]
The block has shape [10, 2048, 2048]

```

Have:

```

Starting example : LazyExamples.lazyLarge1
The image has shape [2048, 2048]
The block has shape [10, 2048, 2048]

```